examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Remove cell?, add sentence explaining what is being plotted below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>